src/core/backend_context.h

```diff
@@ -78,8 +86,9 @@ struct BackendContext {
   int max_batch_size_;
 
 #ifdef TRTIS_ENABLE_GPU
-  // The stream where data transfer operations are executed.
-  // nullptr if the context is created on CPU.
+  // The stream where data transfer operations are executed on.
+  // It must be set explicitly after the context instance is created,
+  // the default is nullptr.
   cudaStream_t stream_;
```

Comment:
I only see a stream created for libtorch backend (except plan which has always created one). I assume other backends will do the same once they support tensors on GPU?

Reply:
I should have done the same for all backends, will do that. And I think all backend contexts have to create its own stream even if they only support tensors on CPU, since the input data may be in GPU and they need to do a device-to-host copy in that case.